src/js/components/Tree/Tree.js

```diff
@@ -0,0 +1,161 @@
+import React, { useContext, useState, useEffect } from 'react';
+import { ThemeContext } from 'styled-components';
+import { FormDown } from 'grommet-icons/icons/FormDown';
+import { FormNext } from 'grommet-icons/icons/FormNext';
+import { FormPrevious } from 'grommet-icons/icons/FormPrevious';
+
+import { Box } from '../Box';
+import { Button } from '../Button';
+import { List } from '../List';
+import { Text } from '../Text';
+
+const Tree = ({ children, mode, data }) => {
+  const theme = useContext(ThemeContext);
+
+  const [depth, setDepth] = useState(0);
```

Comment:
Does depth signify the deepest depth of the data? How does "depth" relate to "level"? The two sound similar. Perhaps a comment?

Reply:
Yep, not obvious ondeed. I added more descriptions and comments about the state variables and how they are related to each other.